docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+# Our entry into our docker development environment is described here.
+
+# For an overview of how to use the Docker Development Environment see `DOCKER.md`
```

Comment:
👍, very happy to see a link to the docs from here, I'd definitely notice this file and it would lead me the right place